Tests/AppTests/AppTests.swift

```diff
@@ -33,13 +36,16 @@ final class AppTests: XCTestCase {
         )
 
         let entries = ChangelogSection.makeSections(from: AppTests.fakeCommits, for: release)
-        XCTAssertEqual(entries.count, 2)
+        XCTAssertEqual(entries.count, 3)
         XCTAssertEqual(entries[0].board, "CNSMR")
-        XCTAssertEqual(entries[0].commits.map { $0.message }, ["[CNSMR-2044] Commit 1", "[CNSMR-2045] Commit 2"])
-        XCTAssertEqual(entries[0].commits.map { $0.ticket?.key }, ["CNSMR-2044", "CNSMR-2045"])
-        XCTAssertEqual(entries[1].board, "CRP")
-        XCTAssertEqual(entries[1].commits.map { $0.message }, ["[CRP-4141] Commit 3", "[CRP-4142] Commit 4"])
-        XCTAssertEqual(entries[1].commits.map { $0.ticket?.key }, ["CRP-4141", "CRP-4142"])
+        XCTAssertEqual(entries[0].commits.map { $0.message }, ["[CNSMR-2044] Commit 1", "[CNSMR-1763] Migrate sdk-nhsgp into sdk-core (#8163)", "[CNSMR-2045] Commit 3"])
+        XCTAssertEqual(entries[0].commits.map { $0.ticket?.key }, ["CNSMR-2044", "CNSMR-1763", "CNSMR-2045"])
+        XCTAssertEqual(entries[1].board, "SDK")
+        XCTAssertEqual(entries[1].commits.map { $0.message }, ["[SDK-4142] Commit 2"])
+        XCTAssertEqual(entries[1].commits.map { $0.ticket?.key }, ["SDK-4142"])
+        XCTAssertEqual(entries[2].board, nil)
+        XCTAssertEqual(entries[2].commits.map { $0.message }, ["Merge remote-tracking branch 'origin/release/babylon/4.4.0' into develop"])
+        XCTAssertEqual(entries[2].commits.map { $0.ticket?.key }, [nil])
 
         let jiraBaseURL = URL(string: "https://babylonpartners.atlassian.net:443")!
```

Comment:
wait, are we making a real api call in the tests?

Reply:
We are not. But since this base URL ends up being used/inserted inside the JSON generated (to embed links to tickets), I ended up using the real URL here just because that allowed me to then provide a test attachment to the test case with the JSON exactly as it would look like… which was then useful to me when testing/validating *manually* with `curl` that it worked and that the JSON jira doc was interpreted correctly as valid markup, when I worked on the PR formatting that JSON.

We could put back a dummy URL in the tests now if you prefer, but if on the future we want to change the formatting of the "Jira document" we generate and we need to run some manual tests via `curl` and the terminal to validate the formatting, it would still be useful to have the real URL i the JSON fixture and be able to just copy-paste it into our `curl` invocation